Gemfile

```diff
@@ -85,7 +85,7 @@ gem "inifile",                        "~>3.0",     :require => false
 gem "logging",                        "~>1.8",     :require => false  # Ziya depends on this
 gem "net_app_manageability",          ">=0.1.0",   :require => false
 gem "net-ping",                       "~>1.7.4",   :require => false
-gem "net-ssh",                        "~>2.9.2",   :require => false
+gem "net-ssh",                        "= 4.0.0.alpha4",   :require => false
```

Comment:
I agree @gmcculloug, much of our deep use of net/ssh isn't thoroughly tested especially not in travis because it's hard. Right?